packages/api/src/lib/structures/http/HttpCodes.ts

```diff
@@ -0,0 +1,427 @@
+	 */
+	RequestTimeout = 408,
+
+	/**
+	 * Indicates that the request could not be processed because of conflict in
+	 * the current state of the resource, such as an edit conflict between
+	 * multiple simultaneous updates.
+	 */
+	Conflict = 409,
+
+	/**
+	 * Indicates that the resource requested is no longer available and will not
+	 * be available again. This should be used when a resource has been
+	 * intentionally removed and the resource should be purged. Upon receiving a
+	 * 410 status code, the client should not request the resource in the future.
+	 * Clients such as search engines should remove the resource from their
+	 * indices. Most use cases do not require clients and search engines to
+	 * purge the resource, and a "404 Not Found" may be used instead.
+	 */
+	Gone = 410,
+
+	/**
+	 * The request did not specify the length of its content, which is required
+	 * by the requested resource.
+	 */
+	LengthRequired = 411,
+
+	/**
+	 * (RFC 7232) The server does not meet one of the preconditions that the
+	 * requester put on the request header fields.
+	 */
+	PreconditionFailed = 412,
+
+	/**
+	 * (RFC 7231) The request is larger than the server is willing or able to
+	 * process. Previously called "Request Entity Too Large".
+	 */
+	PayloadTooLarge = 413,
+
+	/**
+	 * (RFC 7231) The URI provided was too long for the server to process. Often
+	 * the result of too much data being encoded as a query-string of a GET
+	 * request, in which case it should be converted to a POST request. Called
+	 * "Request-URI Too Long" previously.
+	 */
+	URITooLong = 414,
+
+	/**
+	 * (RFC 7231) The request entity has a media type which the server or
+	 * resource does not support. For example, the client uploads an image as
+	 * image/svg+xml, but the server requires that images use a different format.
+	 */
+	UnsupportedMediaType = 415,
+
+	/**
+	 * (RFC 7233) The client has asked for a portion of the file (byte serving),
+	 * but the server cannot supply that portion. For example, if the client
+	 * asked for a part of the file that lies beyond the end of the file. Called
+	 * "Requested Range Not Satisfiable" previously.
+	 */
+	RangeNotSatisfiable = 416,
+
+	/**
+	 * The server cannot meet the requirements of the Expect request-header
+	 * field.
+	 */
+	ExpectationFailed = 417,
+
+	/**
+	 * (RFC 2324, RFC 7168) This code was defined in 1998 as one of the
+	 * traditional IETF April Fools' jokes, in RFC 2324, Hyper Text Coffee Pot
+	 * Control Protocol, and is not expected to be implemented by actual HTTP
+	 * servers. The RFC specifies this code should be returned by teapots
+	 * requested to brew coffee. This HTTP status is used as an Easter egg in
+	 * some websites, such as Google.com's I'm a teapot easter egg.
+	 */
+	IAmATeapot = 418,
+
+	/**
+	 * (RFC 7540) The request was directed at a server that is not able to
+	 * produce a response (for example because of connection reuse).
+	 */
+	MisdirectedRequest = 421,
```

Comment:
```suggestion

	/**
	 * Returned by the Twitter Search and Trends API when the client is being rate limited.
	 * The text is a quote from 'Demolition Man' and the '420' code is likely a reference
	 * to this number's association with marijuana. Other services may wish to implement
	 * the 429 Too Many Requests response code instead.
	 */
	EnhanceYourCalm = 420,

	/**
	 * (RFC 7540) The request was directed at a server that is not able to
	 * produce a response (for example because of connection reuse).
	 */
	MisdirectedRequest = 421,
```

Come on now :^)